### ISSUE TITLE
training model

### ISSUE THREAD

**Issue by Changyiyu** [Xilinx/QNN-MO-PYNQ#35]
Hello,
Can I get the training model of dorefanet-classifier?   I want to train the plant classes.

**Comment by ussamazahid96** [comment]
you should take a look at this [repo](https://github.com/mohdumar644/QNN-MO-PYNQ)

**Comment by KaTaiHo** [comment]
@ussamazahid96 were you able to successfully train new weights for QNN? If so, could you please share the steps to do so? I have looked at the repo that you mentioned above but couldn't get the npz files needed etc.

**Comment by mohdumar644** [comment]
@KaTaiHo did you follow the steps at https://github.com/mohdumar644/QNN-MO-PYNQ/tree/master/qnn/src/training  README?

**Comment by KaTaiHo** [comment]
@mohdumar644 Sorry I made a mistake in my previous post. I meant to ask if we needed to update the npy files such as tinier-yolo-conv0-W.npy from https://github.com/Xilinx/QNN-MO-PYNQ/blob/master/notebooks/tiny-yolo-image.ipynb.

**Comment by mohdumar644** [comment]
Yes if training custom, they are needed for PS layers, only in QNN-MO-PYNQ (not BNN).

**Comment by Asulr** [comment]
> @KaTaiHo did you follow the steps at https://github.com/mohdumar644/QNN-MO-PYNQ/tree/master/qnn/src/training README?
> 
> If you do not use tensorpack, also see https://github.com/mohdumar644/DoReFaNet-PyTorch

Hi, sorry to bother you. I have viewed the comments above and I have ran the DoreFaNet-PyTorch,and got the npz file. But when I use bin-gen from https://github.com/mohdumar644/QNN-MO-PYNQ/tree/master/qnn/src/training, I was failed. There don't have the file 0.npy.

**Comment by B1D1ng** [comment]
> > @KaTaiHo did you follow the steps at https://github.com/mohdumar644/QNN-MO-PYNQ/tree/master/qnn/src/training README?
> > If you do not use tensorpack, also see https://github.com/mohdumar644/DoReFaNet-PyTorch
> 
> Hi, sorry to bother you. I have viewed the comments above and I have ran the DoreFaNet-PyTorch,and got the npz file. But when I use bin-gen from https://github.com/mohdumar644/QNN-MO-PYNQ/tree/master/qnn/src/training, I was failed. There don't have the file 0.npy.

Hi! sorry to bother you. Have you solved this problem and succeed training your custom dataset?

**Comment by B1D1ng** [comment]
> @KaTaiHo did you follow the steps at https://github.com/mohdumar644/QNN-MO-PYNQ/tree/master/qnn/src/training README?
> 
> If you do not use tensorpack, also see https://github.com/mohdumar644/DoReFaNet-PyTorch

Hi, sorry to bother you. If I want to train my own custom dataset for the QNN-MO-PYNQ, I should do the steps in this README?
